src/components/Settings/SettingsPanel/SettingsPlayersSections/SettingsPlayersSections.tsx

```diff
@@ -48,6 +86,7 @@ export default function SettingsPlayersSections(): JSX.Element {
 
             <div className={classes.SettingsInputContainer}>
               <input
+                disabled={list.title === "main.lists.dead"}
```

Comment:
All section names should be editable since we using the icons to indicate the features of certain sections.

Reply:
overlooked, will change